packages/babel-plugin-transform-modules-commonjs/test/fixtures/misc/undefined-this-computed-class-key/output.js

```diff
@@ -0,0 +1,13 @@
+"use strict";
+
+Object.defineProperty(exports, "__esModule", {
+  value: true
+});
+exports.C = void 0;
+
+class C {
+  [this.name]() {}
```

Comment:
It should be overwritten by `rewriteThis`. You can check if `skipAllButComputedKey` is called in the `Function` visitor, which includes `ClassMethod`.

Reply:
@JLHwung Oh! I did not see the `!` before `isArrowFunctionExpression` and thus, didn't change `path.skip()`. Have changed it now to use the `skipAllButComputedKey` function. Also, have re-added the tests for `class-properties`. PTAL.